client/routes.js

```diff
@@ -65,6 +66,10 @@ const routes = [
       {
         path: '/error',
         component: Error
+      },
+      {
+        path: '**',
```

Comment:
In react-router-dom@4 you don't need to specify path. Just be sure that routes are placed inside the Switch component